webview-ui/src/components/settings/AutoApproveSettings.tsx

```diff
@@ -42,73 +44,123 @@ type AutoApproveSettingsProps = HTMLAttributes<HTMLDivElement> & {
 	>
 }
 
-export const AutoApproveSettings = ({
-	alwaysAllowReadOnly,
-	alwaysAllowReadOnlyOutsideWorkspace,
-	alwaysAllowWrite,
-	alwaysAllowWriteOutsideWorkspace,
-	writeDelayMs,
-	alwaysAllowBrowser,
-	alwaysApproveResubmit,
-	requestDelaySeconds,
-	alwaysAllowMcp,
-	alwaysAllowModeSwitch,
-	alwaysAllowSubtasks,
-	alwaysAllowExecute,
-	allowedCommands,
+export const AutoApproveSettings: React.FC<AutoApproveSettingsProps> = ({
```

Comment:
Do you mind pulling any other enhancements out into another PR? Ideally this one would be strictly translations.